nc/source/component/Collider.cpp

```diff
@@ -58,12 +58,13 @@ namespace
 
 namespace nc
 {
-    Collider::Collider(Entity entity, SphereProperties properties)
+    Collider::Collider(Entity entity, SphereProperties properties, bool isTrigger)
         : ComponentBase(entity),
           m_info{.type = ColliderType::Sphere,
                  .offset = properties.center,
                  .scale = Vector3::Splat(properties.radius * 2.0f),
-                 .assetPath = ""}
+                 .assetPath = "",
```

Comment:
This caught me up - I didn't know what type of asset this represented (you had mentioned also storing tensor data in an asset file). Can you name this meshPath? We should assume all paths to resources are assets, so asset isn't needed.

Reply:
Yup, although I changed it to hullAssetPath, as it is not actually a mesh, and mesh asset/ mesh path wording is used elsewhere.